src/index.js

```diff
@@ -13,26 +13,25 @@ export type Props = {
   onAfterRender: () => void,
   onFailure: () => void,
   protocol: string,
-  injectScript: boolean
```

Comment:
Why did you remove? It is become breaking changes.

Reply:
Thank you for review.
I'll rewrite to remove breaking changes.